db/schema.rb

```diff
@@ -69,12 +69,13 @@
   add_index "class_licenses", ["license_id", "class_name"], name: "index_class_licenses_on_license_id_and_class_name", unique: true, using: :btree
 
   create_table "collaborator_solutions", force: :cascade do |t|
-    t.integer  "question_id",   null: false
+    t.integer  "question_id",                                    null: false
     t.string   "title"
-    t.text     "solution_type", null: false
-    t.text     "content",       null: false
-    t.datetime "created_at",    null: false
-    t.datetime "updated_at",    null: false
+    t.text     "solution_type",                                  null: false
+    t.text     "content",                                        null: false
+    t.datetime "created_at",                                     null: false
+    t.datetime "updated_at",                                     null: false
+    t.string   "type",          default: "CollaboratorSolution", null: false
```

Comment:
Uh this shouldn't be here... try `rake db:drop db:create db:migrate db:seed`? You will lose all local data.